cms/urls.py

```diff
@@ -181,6 +184,13 @@
             'contentstore.views.certificates.certificates_list_handler')
     )
 
+urlpatterns += (
+    # Drops into the Programs authoring app, which handles its own routing.
+    # The view uses a configuration model to determine whether or not to
+    # display the authoring app. If disabled, a 404 is returned.
+    url(r'^program/*', ProgramAuthoringView.as_view(), name='programs'),
```

Comment:
I don't think you want the `*` here -- leaving the trailing slash with no `$` will match the URLs you want.